<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formreg</class>
 <widget class="QDialog" name="formreg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1018</width>
        <height>1018</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>111</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/images/images/stureg.webp);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>10</y>
            <width>281</width>
            <height>81</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#5555ff;&quot;&gt;Student Registration Form&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>10</y>
            <width>81</width>
            <height>71</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/images/images/userprof.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="imagebutn">
          <property name="geometry">
           <rect>
            <x>469</x>
            <y>90</y>
            <width>111</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>UPLOAD IMAGE</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>18</x>
            <y>160</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Firstname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="first_name">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>301</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#first_name{
 border: 2px solid blue;
border-radius: 10px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Lastname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lastname">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#lastname{
  border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="middlname">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>370</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#middlname{
   border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Middlename&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="maleradio">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>430</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Male</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="female_radio">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>430</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="other">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>430</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Other</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="mobilenum">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>500</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#mobilenum{
  border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>460</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Mobile no.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>550</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="email">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>590</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#email{
border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>650</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Contact Address&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="contact">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>690</y>
            <width>311</width>
            <height>91</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#contact{
border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>790</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#5555ff;&quot;&gt;Class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="class_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>830</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#class_2{
   border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Register_buttn">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>910</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#Register_buttn{
  border: 2px solid blue;
border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>REGISTER</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
